docs/testplan/build_2tier_network.md

```diff
@@ -0,0 +1,386 @@
+
+```json
+"PORT": {
+    "Ethernet0": {
+        "admin_status": "up",
+        "alias": "Ethernet1/1",
+        "description": "Ethernet1/1",
+        "fec": "rs",
+        "index": "1",
+        "lanes": "17",
+        "mtu": "9100",
+        "pfc_asym": "off",
+        "speed": "100000",
+        "subport": "1",
+        "tpid": "0x8100"
+    }
+},
+"INTERFACE": {
+    "Ethernet0": {},
+    "Ethernet0|2001:db9::1:0:10/120": {}
+}
+```
+
+T1-A Configuration:
+
+```json
+"PORT": {
+    "Ethernet256": {
+        "admin_status": "up",
+        "alias": "Ethernet33/1",
+        "description": "Ethernet33/1",
+        "fec": "rs",
+        "index": "33",
+        "lanes": "273",
+        "mtu": "9100",
+        "pfc_asym": "off",
+        "speed": "100000",
+        "subport": "1",
+        "tpid": "0x8100"
+    }
+},
+"INTERFACE": {
+    "Ethernet256": {},
+    "Ethernet256|2001:db9::1:0:1/120": {}
+}
+```
+
+#### Specify BGP Neighbor Attributes
+
+Each switch must recognize the other as a BGP neighbor and establish a peer session for route exchange.
+
+T0-A Configuration:
+
+```json
+"BGP_NEIGHBOR": {
+    "2001:db9::1:0:10": {
+        "admin_status": "up",
+        "asn": 65001,
+        "holdtime": "10",
+        "keeplive": "3",
+        "local_addr": "2001:db9::1:0:1",
+        "name": "T1-A-0",
+        "nhopself": "0",
+        "rrclient": "0"
+    }
+},
+"DEVICE_NEIGHBOR": {
+    "Ethernet256": {
+        "name": "T1-A-0",
+        "port": "Ethernet256"
+    }
+},
+"DEVICE_NEIGHBOR_METADATA": {
+    "T1-A-0": {
+        "cluster": "StressTest",
+        "deployment_id": "1",
+        "lo_addr": "10.100.0.88/32",
+        "lo_addr_v6": "fc00:1::88/128",
+        "mgmt_addr": "10.64.246.88/23",
+        "mgmt_addr_v6": "fc00:2::88/64",
+        "hwsku": "ABCDEFG",
+        "type": "LeafRouter"
+    }
+}
+```
+
+T1-A Configuration:
+
+```json
+"BGP_NEIGHBOR": {
+    "2001:db9::1:0:1": {
+        "admin_status": "up",
+        "asn": 64001,
+        "holdtime": "10",
+        "keeplive": "3",
+        "local_addr": "2001:db9::1:0:10",
+        "name": "T0-A-0",
+        "nhopself": "0",
+        "rrclient": "0"
+    }
+},
+"DEVICE_NEIGHBOR": {
+    "Ethernet0": {
+        "name": "T0-A-0",
+        "port": "Ethernet0"
+    }
+},
+"DEVICE_NEIGHBOR_METADATA": {
+    "T0-A-0": {
+        "cluster": "StressTest",
+        "deployment_id": "1",
+        "lo_addr": "10.100.0.81/32",
+        "lo_addr_v6": "fc00:1::81/128",
+        "mgmt_addr": "10.64.246.81/23",
+        "mgmt_addr_v6": "fc00:2::81/64",
+        "hwsku": "ABCDEFG",
+        "type": "ToRRouter"
+    }
+}
+```
+
+### Verifying  BGP Establishment
+
+Once the configurations are applied, the BGP session will be established, and the switches will begin exchanging routing information. Proper verification using `show bgp summary` and `show bgp neighbors` commands can be conducted to confirm session establishment and route propagation.
+
+## Establish Scaling Number of Routes
```

Comment:
the "large scale" sounds like the test setup for route scale test, which is not really required.

in here, we only need to make sure traffic generator can advertise the routes, so in the test, we can advertise any route that we need.

most of the time, the route is advertised per logical port, so we can easily write a fixture to advertise X number of routes for each logical port.